ffi/dotnet/Devolutions.IronRdp/src/Exceptions.cs

```diff
@@ -1,11 +1,13 @@
+namespace Devolutions.IronRdp;
+
 [Serializable]
 public class IronRdpLibException : Exception
 {
-    public IronRdpLibExceptionType type { get; private set; }
+    public IronRdpLibExceptionType Type { get; private set; }
```

Comment:
nit: I'd be really inclined to call this "ExceptionType" or "ErrorType" or something, just because `Type` is overloaded. That's just my taste, though 👅